README.md

```diff
@@ -0,0 +1,174 @@
+# MCS Benchmark Data
+
+MCS Benchmark Data is a set of scripts that can be ued to create and execute Extract, Transform, Load (ETL) pipelines that store information about Machine Commonsense benchmarks into the Commonsense Benchmark Ontology.
+
+For more information about the ontology, refer to:
+[Commonsense Benchmark Ontology](https://tetherless-world.github.io/mcs-ontology/docs/index-en.html)
+*Henrique O. Santos, Minor Gordon, Zhicheng Liang, Gretchen Forbush, Deborah L. McGuinness*
+
+
+## Usage: Adding a Pipeline

```

Comment:
This is really development rather than usage. Typically it's Use in a section and then Development in another section.

Reply:
See the mowgli-etl repository for a decent example.